packages/klighd-core/src/options/options-module.ts

```diff
@@ -35,6 +36,9 @@ export const optionsModule = new ContainerModule((bind, _, isBound) => {
     bind(GeneralPanel).toSelf().inSingletonScope();
     bind(DISymbol.SidebarPanel).toService(GeneralPanel);
 
+    bind(BookmarkPanel).toSelf().inSingletonScope();
+    bind(DISymbol.SidebarPanel).toService(BookmarkPanel);
+
```

Comment:
The panel should be registered in the bookmark module, since it belongs to the module.

Reply:
has been moved to the bookmark module